builtin/credential/aws-ec2/path_config_client.go

```diff
@@ -1,4 +1,4 @@
-package aws
+package awsEc2
```

Comment:
Package names in Go are always lowercase, that I've ever seen. Is there a reason for the E to be capitalized?